templates/parts/listing-title.tpl.php

```diff
@@ -6,12 +6,22 @@
  */
 ?>
 <?php if ( $title_type !== 'h1' ) : ?>
-	<div class="listing-title">
+	<?php
+		$class_name = 'listing-title';
+
+		if ( $title_type === true ) {
+			$title_type = 'h1';
+			$class_name .= ' show-listing-title';
+		}
+	?>
+	<div class="<?php echo esc_attr( 'listing-title' . $class_name ); ?>">
```

Comment:
Will the class end up looking like this?
"listing-titlelisting-title show-listing-title" or "listing-titlelisting-title"

Reply:
Yes, exactly.

Reply:
From what I can see, listing-title is duplicated and there is a space missing. Are you seeing this too?

Reply:
Oh, my bad!